PelotonEppSdk/Models/EventResponse.cs

```diff
@@ -75,13 +78,21 @@ internal class event_response : response
             if (eventResponse.items != null)
                 items = eventResponse.items.Select(ei => (EventItem)ei).ToList();
 
+            var eventState = (State)eventResponse.state;
+            EventStateEnum eventStateEnumValue = EventStateEnum.Unknown;
```

Comment:
nit:
I don't know if this is better, but I feel that removing the if would clarify the intent a bit better

var eventStateEnumValue = (eventState == null)
  ?  EventStateEnum.Unknown
  : (EventStateEnum) int.Parse(eventState.Code);